src/Microsoft.AspNetCore.NodeServices/Microsoft.AspNetCore.NodeServices.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Invoke Node.js modules at runtime in ASP.NET Core applications.</Description>
+    <VersionPrefix>1.1.1-alpha</VersionPrefix>
```

Comment:
Some of these projects are RTM and some are not, so they probably don't all version the same.